docs/desktop-feature-api.md

```diff
@@ -6,11 +6,22 @@ slug: /desktop-feature-api
 
 This guide will help you use the Nimbus Feature API in Desktop Firefox to run experiments, set values remotely, and manage user preferences. If you are familiar with Normandy and are trying to migrate a feature, you may want to check out the [Migration Guide for Pref Experiments](desktop-migration-guide).
 
-:::info
-Your code must be able to import `ExperimentAPI.jsm` in the parent process or a privileged child process. We do not currently support a C++/Rust API.
-:::
+### API changes in Firefox 90
+
+- 🆕 Added a [`getVariable()`](#getvariable) API that optimizes accessing single variables.
+- 🆕 Added an `isEarlyStartup` option in the manifest. Use this if you need to cache values for early synchronous access.
+- ⚠️ Deprecated `getValue()` method in favour of [`getAllVariables()`](#getvariable). Please use this for new code.
```

Comment:
nit: ```[`getAllVariables()`](#getvariable)``` feels like something is wrong. The URL anchor doesn't seem to match the text, and I am wondering if the anchor should be `#getallvariables`, or if the linked text should be `getVariable()`.